src/core/identifiers/identifiers.ts

```diff
@@ -85,8 +86,9 @@ export const convertDSNPUserIdOrURIToBigNumber = (userIdOrUri: DSNPUserId | DSNP
  * @param userURI - The DSNPUserURI to convert
  * @returns The DSNPUserId of the user
  */
-export const convertDSNPUserURIToDSNPUserId = (userURI: DSNPUserURI): DSNPUserId =>
-  userURI.replace(/dsnp:\/\/0x0+/, "0x0");
+export const convertDSNPUserURIToDSNPUserId = (userURI: DSNPUserURI): DSNPUserId => {
+  return serializeToHex(userURI);
+};
```

Comment:
if this function is just an alias for `serializeToHex`, that makes me think we should delete one and just use the other

Reply:
There are a few of these. I wanted to keep them for backward compatibility and also for better typing and naming.

Reply:
I want the specific types however.